soc/arm/nxp_kinetis/k2x/Kconfig.series

```diff
@@ -0,0 +1,16 @@
+# Kconfig - Kinetis K2X MCU line
+#
+# Copyright (c) 2018 Prevas A/S
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_SERIES_KINETIS_K2X
+	bool "Kinetis K2x Series MCU"
+	select CPU_CORTEX_M4
+	select SOC_FAMILY_KINETIS
+	select SYS_POWER_LOW_POWER_STATE_SUPPORTED
+	select CPU_HAS_SYSTICK
```

Comment:
Please add `select CPU_HAS_NXP_MPU` as in `soc/arm/nxp_kinetis/k6x/Kconfig.series`

Reply:
K22P121M120SF7RM.pdf mentions nothing about a memory protection unit...
The Kinetis K22 MCU Family Block Diagram have it as optional for K22